CONTRIBUTING.md

```diff
@@ -54,19 +54,12 @@ Unsure where to begin contributing to Grafana? Start by browsing issues labeled
 
 If you're looking to make a code change, see how to set up your environment for [local development](contribute/developer-guide.md).
 
-When you're ready to contribute, it's time to [Create a pull request](/contribute/pull-request.md).
+When you're ready to contribute, it's time to [Create a pull request](/contribute/create-pull-request.md).
 
 #### Contributor License Agreement (CLA)
 
 Before we can accept your pull request, you need to [sign our CLA](https://grafana.com/docs/contribute/cla/). If you haven't, our CLA assistant prompts you to when you create your pull request.
 
-## Community
-
-- Follow [@grafana on Twitter](https://twitter.com/grafana/)
-- Read and subscribe to the [Grafana blog](https://grafana.com/blog/)
-- If you have a specific question, check out our [discussion forums](https://community.grafana.com).
-- For general discussions, join us on the [official Slack](http://slack.raintank.io/).
-
```

Comment:
Why remove this section completely? Is getting rewritten and added somewhere else?

Reply:
I added it to the README.md, under "Get involved". Is it worth duplicating?

Reply:
That's fine - just wondering.